extensions/wikia/Email/Controller/FounderController.class.php

```diff
@@ -0,0 +1,211 @@
+<?php
+
+namespace Email\Controller;
+
+use Email\Check;
+use Email\EmailController;
+use Email\Fatal;
+
+abstract class FounderController extends EmailController {
```

Comment:
I saw this in the original but I thought some how the ticket didn't specify this level of tracking originally.  I'll look into it ...